micrometer/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector
+    command: [--config=/etc/otel-collector-config.yml]
```

Comment:
```
emartins@emartins-mac micrometer % docker-compose up
[+] Building 0.0s (0/0)                                    docker:desktop-linux
[+] Running 2/2
 ✔ Network micrometer_default             Created                          0.0s 
 ✔ Container micrometer-otel-collector-1  Created                          0.0s 
Attaching to micrometer-otel-collector-1
micrometer-otel-collector-1  | Error: failed to resolve config: cannot resolve the configuration: cannot retrieve the configuration: unable to read the file file:/etc/otel-collector-config.yml: read /etc/otel-collector-config.yml: is a directory
micrometer-otel-collector-1  | 2024/01/08 14:25:55 collector server run finished with error: failed to resolve config: cannot resolve the configuration: cannot retrieve the configuration: unable to read the file file:/etc/otel-collector-config.yml: read /etc/otel-collector-config.yml: is a directory
micrometer-otel-collector-1 exited with code 1
```
My guess is the .yml instead of .yaml on this file too